.appveyor.yml

```diff
@@ -24,65 +24,31 @@ clone_depth: 50    #same as travis, see https://www.appveyor.com/blog/2014/06/04
-# https://www.appveyor.com/docs/build-cache/#cleaning-up-cache
-
+    - c:\Tools\vcpkg\installed
 
 environment:
-    openssl_ver: 1.0.2p
-    protobuf_ver: 3.6.1
-    zlib_ver: 1.2.11
-
     matrix:
         - target_arch: win64
           qt_ver: 5.9\msvc2017_64
           cmake_generator: Visual Studio 15 2017 Win64
           cmake_toolset: v141,host=x64
-          vc_arch: amd64
+          vcpkg_arch: x64
 
         - target_arch: win32
           qt_ver: 5.9\msvc2015 # Qt doesn't provide a msvc2017_32
           cmake_generator: Visual Studio 15 2017
           cmake_toolset: v141
-          vc_arch: amd64_x86
-
+          vcpkg_arch: x86
 
 install:
     - ps: |
-        if (Test-Path c:\openssl-release) {
-            echo "using openssl from cache"
-        } else {
-            if ($env:target_arch -eq "win64") {    # 64bit filename
-                # echo "downloading 64bit version of openssl"
-                Invoke-WebRequest "https://indy.fulgan.com/SSL/openssl-$env:openssl_ver-x64_86-win64.zip" -OutFile c:\openssl-$env:openssl_ver.zip
-            } else {    # 32bit filename
-                # echo "downloading 32bit version of openssl"
-                Invoke-WebRequest "https://indy.fulgan.com/SSL/openssl-$env:openssl_ver-i386-win32.zip" -OutFile c:\openssl-$env:openssl_ver.zip
-            }
-            Expand-Archive -Path c:\openssl-$env:openssl_ver.zip -DestinationPath c:\openssl-release
-            Set-Location -Path C:\openssl-release
-        }
-        if (Test-Path c:\protobuf-release) {
-            echo "using protobuf from cache"
-        } else {
-            Invoke-WebRequest "https://github.com/protocolbuffers/protobuf/releases/download/v$env:protobuf_ver/protobuf-cpp-$env:protobuf_ver.zip" -OutFile c:\protobuf-cpp-$env:protobuf_ver.zip
-            Expand-Archive -Path c:\protobuf-cpp-$env:protobuf_ver.zip -DestinationPath c:\
-            Set-Location -Path C:\protobuf-$env:protobuf_ver\cmake
-            cmake . -G "$env:cmake_generator" -T "$env:cmake_toolset" -Dprotobuf_BUILD_TESTS=0 -Dprotobuf_MSVC_STATIC_RUNTIME=0 -DCMAKE_INSTALL_PREFIX=c:/protobuf-release
-            msbuild INSTALL.vcxproj /p:Configuration=Release
-        }
-        if (Test-Path c:\zlib-release) {
-            echo "using zlib from cache"
+        if (Test-Path C:\Tools\vcpkg\installed\x64-windows) {
+          echo "using vcpkg packages from cache"
         } else {
-            Invoke-WebRequest "https://github.com/madler/zlib/archive/v$env:zlib_ver.zip" -OutFile c:\zlib-$env:zlib_ver.zip
-            Expand-Archive -Path c:\zlib-$env:zlib_ver.zip -DestinationPath c:\
-            Set-Location -Path C:\zlib-$env:zlib_ver
-            cmake . -G "$env:cmake_generator" -T "$env:cmake_toolset" -DCMAKE_INSTALL_PREFIX=c:/zlib-release
-            msbuild INSTALL.vcxproj /p:Configuration=Release
+          vcpkg install openssl:$env:vcpkg_arch-windows
```

Comment:
`vcpkg install` is a no-op if the packages are already installed, so I would instead recommend that you always run ```
vcpkg remove --outdated --recurse
vcpkg install openssl protobuf liblzma zlib --triplet $env:vcpkg_arch-windows
```

The remove call will purge any existing libraries that don't match the current commit ID.

Reply:
Great advice, thank you!